### PR TITLE
bugfix/20662-rangeselector-dropdown-a11y

### LINKED ISSUE #20662: Keyboard trap on rangeSelector dropdown
#### Expected behaviour
To be able to use tab-key in chart and interact with dropdown rangeSelector

#### Actual behaviour
After rangeSelector dropdown is focused, I am not able to tab to next element and focus indicator is stuck. Backwards tabbing seems fine. 

#### Live demo with steps to reproduce
https://jsfiddle.net/maritavindedal/8j71zLkq/

#### Product version
Highcharts Stock

### PR DESCRIPTION
**Pull request by kamil-musialowski**
Fixed #20662, keyboard focus with `Tab` key was not working when `rangeSelector` dropdown was focused.

### CODE REVIEW

**Review by pawelfus — APPROVED** [review]
Looks good, I guess? Requesting @marvin19 for some help here

**Review by DJTechnoo — APPROVED** [review]
Excellent!

**Review by marvin19 — APPROVED** [review]
Working! 👍

**Review by pawellysy — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Comment by kamil-musialowski** [comment]
@TorsteinHonsi Do we merge this one or we're waiting for something? 🤔

**Comment by TorsteinHonsi** [comment]
We were just waiting for the final review. Thanks!

**Pull request merged**
